lib/active_merchant/billing/gateways/paysafe.rb

```diff
@@ -26,6 +26,7 @@ def purchase(money, payment, options = {})
         add_three_d_secure(post, payment, options) if options[:three_d_secure]
         add_stored_credential(post, options) if options[:stored_credential]
         add_split_pay_details(post, options)
+        add_funding_transaction(post, options)
```

Comment:
Don't want to make extra work for you, but this might be a nice opportunity for some refactoring with a `add_auth_purchase_params` method since the list of added methods is getting pretty long in auth and purchase, just a thought.

Reply:
Good call! I pushed a change that extracts the shared logic out into `add_auth_purchase_params`. There were a couple of things that only existed in purchase and not authorize, so I left them out of the new method, in case they were purchase-specific.